apis/github/tool.gpt

```diff
@@ -166,3 +166,11 @@ Param: owner: the owner of the repository
 Param: repo: the name of the repository
 
 #!/usr/bin/env node ${GPTSCRIPT_TOOL_DIR}/index.js getStarCount
+
+---
+!metadata:*:category
+GitHub
+
+---
+!metadata:*:icon
+https://raw.githubusercontent.com/FortAwesome/Font-Awesome/refs/tags/6.5.2/svgs/brands/github.svg
```

Comment:
It seems like the react icons site won't give you a direct URL to the icon SVG -- please correct me if I'm wrong here -- so I went to the source to get these.

Reply:
Any reason to prefer this one over the simpleicons one?

Reply:
I tried to match the icons to what we had in desktop, but it probably makes more sense to just use simpleicons everywhere.

Reply:
done